1-js/02-first-steps/09-comparison/1-comparison-questions/task.md

```diff
@@ -1,10 +1,10 @@
-importance: 5
+اهمیت: 5
```

Comment:
Metadata shouldn't be translated. [[+](https://github.com/javascript-tutorial/fa.javascript.info#metadata)]